Library/Homebrew/cask/download.rb

```diff
@@ -45,17 +47,13 @@ def time_file_size(timeout: nil)
       downloader.resolved_time_file_size(timeout: timeout)
     end
 
-    def clear_cache
-      downloader.clear_cache
-    end
+    def_delegator :downloader, :clear_cache, :clear_cache
```

Comment:
```suggestion
    def_delegators :downloader, :clear_cache, :basename
```

Although may want to move elsewhere.